neo/rawio/neuronexusrawio.py

```diff
@@ -0,0 +1,285 @@
+            )
+
+        # organize the channels
+        signal_channels = []
+        channel_info = self.metadata["sapiens_base"]["biointerface_map"]
+
+        # as per dicussion with the Neo/SpikeInterface teams stream_id will become buffer_id
+        # and because all data is stored in the same buffer stream for the moment all channels
+        # will be in stream_id = 0. In the future this will be split into sub_streams based on
+        # type but for now it will be the end-users responsability for this.
+        stream_id = '0'  # hard-coded see note above
+        for channel_index, channel_name in enumerate(channel_info["chan_name"]):
+            channel_id = channel_info["ntv_chan_name"][channel_index]
+            # 'ai0' indicates analog data which is stored as microvolts
+            if channel_info["chan_type"][channel_index] == "ai0":
+                units = "uV"
+            # 'd' means digital. Per discussion with neuroconv users the use of
+            # 'a.u.' makes the units clearer
+            elif channel_info["chan_type"][channel_index][0] == "d":
+                units = "a.u."
+            # aux channel
+            else:
+                units = "V"
+
+            signal_channels.append(
+                (
+                    channel_name,
+                    channel_id,
+                    self._sampling_frequency,
+                    BINARY_DTYPE,
+                    units,
+                    1,  # no gain
+                    0,  # no offset
+                    stream_id,
+                )
+            )
+
+        signal_channels = np.array(signal_channels, dtype=_signal_channel_dtype)
+
+        stream_ids = np.unique(signal_channels["stream_id"])
+        signal_streams = np.zeros(stream_ids.size, dtype=_signal_stream_dtype)
+        signal_streams["id"] = [str(stream_id) for stream_id in stream_ids]
+        for stream_index, stream_id in enumerate(stream_ids):
+            name = stream_id_to_stream_name.get(int(stream_id), "")
+            signal_streams["name"][stream_index] = name
+
+        # No events
+        event_channels = []
+        event_channels = np.array(event_channels, dtype=_event_channel_dtype)
+
+        # No spikes
+        spike_channels = []
+        spike_channels = np.array(spike_channels, dtype=_spike_channel_dtype)
+
+        # Put all the necessary info in the header
+        self.header = {}
+        self.header["nb_block"] = 1
+        self.header["nb_segment"] = [1]
+        self.header["signal_streams"] = signal_streams
+        self.header["signal_channels"] = signal_channels
+        self.header["spike_channels"] = spike_channels
+        self.header["event_channels"] = event_channels
+
+        # Add the minimum annotations
+        self._generate_minimal_annotations()
+
+        self.header['rec_datetime'] = self.metadata['status']['start_time']
```

Comment:
@zm711 could check this instead ?
the rec_datetime must be a real datatime and this must routed inside raw_annotaion at block and segment level for instance like this

```suggestion
        rec_datetime = datetime.datetime(self.metadata['status']['start_time'])
        bl_annotations = self.raw_annotations["blocks"][0]
        seg_annotations = bl_annotations["segments"][0]
        for d in (bl_annotations, seg_annotations):
            d["rec_datetime"] = rec_datetime
```